docs/manuals/source/man/bareos-dbcopy.rst

```diff
@@ -18,7 +18,7 @@ for both ``<sourcecatalog>`` and ``<destinationcatalog>`` must exist.
 
 The main purpose of :program:`bareos-dbcopy` is to migrate an existing Bareos
 installation from |mysql| to |postgresql|. Therefore the required
-:config:option:`dir/catalog/DbDriver` in the ``<sourcecatalog>`` is |mysql|, and
+``dir/catalog/DbDriver`` in the ``<sourcecatalog>`` is |mysql|, and
```

Comment:
Didn't we remove dbcopy in an earlier PR? Why do we still keep its manpage?
We should definitely still point people at the tool, that existed in prior versions, but I guess it is sufficient if the documentation is present in that version...